### PR TITLE
Adding spacewalk plugin

### PR DESCRIPTION
**Pull request by cocker-cc**
Adding spacewalk plugin.  RPM available [here](https://yum.theforeman.org/plugins/nightly/el7/x86_64/)

### CODE REVIEW

File: manifests/plugin/foreman_wreckingball.pp [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Installs foreman_wreckingball plugin
```
Comment (ekohl) [inline manifests/plugin/foreman_wreckingball.pp]: Looks like you included https://github.com/theforeman/puppet-foreman/pull/647 here. For the changelog it'd be nice to separate the two.

**Comment by cocker-cc** [comment]
I cleaned it up.

**Comment by cocker-cc** [comment]
I add a trivial Test, and renamed the File.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**